Formula/autogen.rb

```diff
@@ -22,17 +22,17 @@ class Autogen < Formula
     sha256 x86_64_linux:   "459c36573772600aab0085300e551ecbbe224a8b036bc10c15d48db1719a5a52"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
You don't appear to be using these. Plus for the guile version check your patch is changing `configure` anyway so you probably don't need to patch the m4 files at all.

Reply:
I'm also modifying `config/guile.m4`, and the build complained it needed `aclocal`.

Reply:
The exact errors I had are:
```
/private/tmp/autogen-20211203-68493-1mj787z/autogen-5.18.16/config/missing: line 81: aclocal-1.16: command not found
'aclocal-1.16' is missing on your system.
sh: autom4te: command not found
```